catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/TestCswEndpoint.java

```diff
@@ -2555,6 +2547,22 @@ public void testUpdateTransactionWithConstraint()
                 .getValue(), is("bar"));
     }
 
+    @Test
+    public void testQueryTags() throws Exception {
+        csw.setSchemaToTagsMapping(new String[] {CswConstants.CSW_OUTPUT_SCHEMA + "=myTag"});
+        GetRecordByIdRequest request = new GetRecordByIdRequest();
```

Comment:
I don't know if it's helpful ( I may not have had to do it, these lines look awful similar) but I made a CSW Query builder helper class in the integration tests when I was testing out my query filter for invalid metacards.

It's called `CswQueryBuilder`, in `distribution/test/itests/test-itest-ddf/common/`